src/Microsoft.ML.Tokenizers/Model/EnglishRoberta.cs

```diff
@@ -592,6 +593,13 @@ public override bool IsValidChar(char ch)
         {
             return _byteToUnicode.ContainsKey(ch);
         }
+
+        public override bool IsFirstTokenInWord(string token)
```

Comment:
>public override bool IsFirstTokenInWord(string token) [](http://example.com/codeflow?start=8&length=53)

nit:

public override bool IsFirstTokenInWord(string token) => token is null || (token.Length != 0 && token[0] == _startChar)